### DISCUSSION TITLE
Label case customization

### DISCUSSION THREAD

**Discussion by arolluisma** [zauberzeug/nicegui#2216] [Q&A] [answered]
### Question

The labels of buttons, tabs, etc. are all caps (uppercase), which seem to be the default. What property should I modify if I want my labels to be in, say, title case? Thanks.

**Comment by falkoschindler** [comment, marked as answer]
You can either set the "no-caps" prop per button
```py
ui.button('Button').props('no-caps')
```

or for _all_ buttons:
```py
ui.button.default_props('no-caps')
ui.button('Button')
```
